recipes/libpqxx/all/conanfile.py

```diff
@@ -67,6 +82,8 @@ def _configure_cmake(self):
             self._cmake = CMake(self)
             self._cmake.definitions["BUILD_DOC"] = False
             self._cmake.definitions["BUILD_TEST"] = False
+            # Set `-mmacosx-version-min` to enable C++17 standard library support.
+            self._cmake.definitions['CMAKE_OSX_DEPLOYMENT_TARGET'] = self._mac_os_minimum_required_version
```

Comment:
that's interesting, isn't it set by conan already?

Reply:
@SSE I thought so too, but the CI failed to build apple-clang 11 (https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/6566/9/libpqxx/7.6.0//summary.json).
Several recipes had this CMake variable set, so I did the same for this recipe and the build succeeded.